src/App.css

```diff
@@ -1,38 +1,572 @@
-.App {
+/* page body */
+
+body {
   text-align: center;
+  background: rgb(255, 254, 143);
+  background: linear-gradient(
+    45deg,
+    rgb(255, 225, 143) 0%,
+    rgba(255, 211, 140, 1) 100%
+  );
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+/* keeps footer at bottom */
+.container {
+  display: flex;
+  min-height: 100vh;
+  flex-direction: column;
+  justify-content: space-between;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
+.mainDiv {
+  margin-top: 100px;
+  position: relative;
+  margin-left: auto;
+  margin-right: auto;
+  width: 80%;
+}
+@media (max-width: 1050px) {
```

Comment:
It's recommended to use `min-width` since most users are on mobile - so they'll get the correct styles first without having to also process the media queries.

Reply:
(Though you could make a pretty good argument for this website being desktop-first...)